portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SegmentsDistribution.testcase

```diff
@@ -145,6 +145,7 @@ definition {
 	@priority = "3"
 	test CanDistributionChartFilteredNumber {
 
+		// CSV data source tests are no longer possible to run due to this ticket LRAC-11651
```

Comment:
Hey @manuferreira, I think this validation can be done using other criteria.

I think you can undo this change and create a ticket for us to refactor this test or you can do it yourself in your ticket